app/server/utils/phone.py

```diff
@@ -41,68 +36,35 @@ def proccess_phone_number(phone_number, region=None, ignore_region=False):
     return parsed_phone_number
 
 
-def send_phone_verification_message(to_phone, one_time_code):
-    if to_phone:
-        reciever_message = 'Your Sempo verification code is: {}'.format(one_time_code)
-
-        send_generic_message(to_phone, reciever_message)
-
-
-def send_onboarding_message(to_phone, first_name, credits, one_time_code):
-    if credits is None:
-        credits = 0
+class ChannelType(enum.Enum):
+    TWILIO = "tw"
+    AFRICAS_TALKING = "at"
+    MESSAGEBIRD = "mb"
 
-    if to_phone:
-        receiver_message = '{}, you have been registered for {}. You have {} {}. Your one-time code is {}. ' \
-                           'Download Sempo for Android: https://bit.ly/2UVZLqf'\
-            .format(
-            first_name,
-            current_app.config['PROGRAM_NAME'],
-            credits,
-            current_app.config['CURRENCY_NAME'],
-            one_time_code,
-            current_app.config['CURRENCY_NAME']
-        )
-
-        send_generic_message(to_phone, receiver_message)
-
-def send_intro_message(to_phone, credits, pin):
-
-    if to_phone:
-        receiver_message = 'You have been registered for {}. You have {} {}. Your PIN is {}. ' \
-                           'To send {} to another phone number type SEND.' \
-                .format(
-            current_app.config['PROGRAM_NAME'],
-            credits/100,
-            current_app.config['CURRENCY_NAME'],
-            pin,
-            current_app.config['CURRENCY_NAME']
-        )
 
-        # send_generic_message(to_phone, receiver_message)
+# just checking by area code may break down one day since multiple countries share the same country codes...
+def channel_for_number(phone):
```

Comment:
might be good to use `phonenumbers.parse(number)` here to extract the country code? they've probably got better edge case management. no strong opinion though.

Reply:
i played around with `phonenumber.parse` but it seemed like you needed to pass in the region as well? otherwise i got an empty region in the returned object. or did i do it wrong?

Reply:
ah i was trying to print `phone_obj.region` and that was empty string, but country_code comes out correctly. will switch to that

Reply:
hmm just tried it with a fake trinidadian phone number and parse outputs country code as just 1. in which case matching by string might actually give us better granularity

Reply:
oh weird! all g then